workspaces/azure-sites/plugins/azure-sites-backend/src/service/router.ts

```diff
@@ -47,16 +43,16 @@ export interface RouterOptions {
   catalogApi: CatalogApi;
   permissions: PermissionsService;
   discovery: DiscoveryService;
-  auth?: AuthService;
-  httpAuth?: HttpAuthService;
+  auth: AuthService;
```

Comment:
mmm how could adopter instantiate this plugin using the legacy backend system? 🤔